src/DynamoCore/Configuration/GraphChecksumItem.cs

```diff
@@ -1,17 +1,26 @@
 using System;
-using System.Collections.ObjectModel;
-using Dynamo.Core;
-using Dynamo.Properties;
+using System.Collections.Generic;
 
 namespace Dynamo.Configuration
 {
     /// <summary>
     /// Represents the stringified version of the nodes connections from a graph
     /// </summary>
+    [Obsolete("This property is not needed anymore in the preference settings and can be removed in a future version of Dynamo.")]
     public class GraphChecksumItem
     {
         public string GraphId { get; set; }
 
         public string Checksum { get; set; }   
     }
+
+    /// <summary>
+    /// Represents the stringified version of the nodes connections from a graph
+    /// </summary>
+    public class GraphChecksumPair
```

Comment:
Looks like we need to add this class to declared API as suggested by Github action